.github/workflows/ci.yaml

```diff
@@ -19,10 +19,10 @@ on:
 env:
   VAULT_ADDR: https://vault.eng.aserto.com/
   PRE_RELEASE: ${{ github.ref == 'refs/heads/main' && 'main' || '' }}
-  GO_VERSION: "1.20"
+  GO_VERSION: "1.22.1"
```

Comment:
I think we would want to pin to 1.22 so we pick up the latest fixes without having to update CI, that is what we did before as well.